### ISSUE TITLE
ReaderCommentCell: crash on configureTime()

### ISSUE THREAD

**Issue by kurzee** [wordpress-mobile/WordPress-iOS#6770]
Likely a bad date value: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Reader/ReaderCommentCell.swift#L163

Could potentially be related and fixed via #6769.

### Version
WPiOS 6.9+

### Log
```
Crashed: com.apple.main-thread
0  WordPress                      0x10050a130 ReaderCommentCell.configureTime() -> () (ReaderCommentCell.swift:168)
1  WordPress                      0x100509b8c @objc ReaderCommentCell.configureCell(comment : Comment) -> () (ReaderCommentCell.swift:124)
2  WordPress                      0x1001adf30 -[ReaderCommentsViewController configureCell:atIndexPath:] (ReaderCommentsViewController.m:817)
3  WordPress                      0x1001ae0b8 -[ReaderCommentsViewController tableView:cellForRowAtIndexPath:] (ReaderCommentsViewController.m:836)
4  WordPress                      0x10017d3f8 -[WPTableViewHandler tableView:cellForRowAtIndexPath:] (WPTableViewHandler.m:347)
5  UIKit                          0x196e09b58 -[UITableView _createPreparedCellForGlobalRow:withIndexPath:willDisplay:] + 716
6  UIKit                          0x196e09d88 -[UITableView _createPreparedCellForGlobalRow:willDisplay:] + 80
7  UIKit                          0x196df7320 -[UITableView _updateVisibleCellsNow:isRecursive:] + 2304
8  UIKit                          0x196e0edec -[UITableView _performWithCachedTraitCollection:] + 116
9  UIKit                          0x196baade8 -[UITableView layoutSubviews] + 176
10 UIKit                          0x196ac3a80 -[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 1196
11 QuartzCore                     0x193f719d8 -[CALayer layoutSublayers] + 148
12 QuartzCore                     0x193f664cc CA::Layer::layout_if_needed(CA::Transaction*) + 292
13 QuartzCore                     0x193f6638c CA::Layer::layout_and_display_if_needed(CA::Transaction*) + 32
14 QuartzCore                     0x193ee33e0 CA::Context::commit_transaction(CA::Transaction*) + 252
15 QuartzCore                     0x193f0aa68 CA::Transaction::commit() + 512
16 QuartzCore                     0x193f0b488 CA::Transaction::observer_callback(__CFRunLoopObserver*, unsigned long, void*) + 120
17 CoreFoundation                 0x190bbe0c0 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 32
18 CoreFoundation                 0x190bbbcf0 __CFRunLoopDoObservers + 372
19 CoreFoundation                 0x190bbc180 __CFRunLoopRun + 1024
20 CoreFoundation                 0x190aea2b8 CFRunLoopRunSpecific + 444
21 GraphicsServices               0x19259e198 GSEventRunModal + 180
22 UIKit                          0x196b317fc -[UIApplication _run] + 684
23 UIKit                          0x196b2c534 UIApplicationMain + 208
24 WordPress                      0x1000f72c0 main (main.m:5)
25 libdispatch.dylib              0x18facd5b8 (Missing)
```

**Comment by aerych** [comment]
> Could potentially be related and fixed via #6769.

Seems likely based on the contents of the logs.  All the reports I've looked at are showing either deleted posts or deleted topics. 

I'm cautiously optimistic its resolved.

**Comment by bummytime** [comment]
I am going to close this — it looks like #6769 probably fixed it. Last occurrence of this crasher was a while ago on devices running iOS9 + WPiOS Version 7.0.2.0.
